alpha/lib/model/DeliveryProfileDynamicAttributes.php

```diff
@@ -477,11 +477,23 @@ public function filterFlavorsByTags($flavors)
 			}
 				
 			if ($curFlavors)
+			{
+				$this->addAltAudioFlavors($flavors, $curFlavors);
 				return $curFlavors;
+			}
 		}
 		return array();
 	}
 
+	private function addAltAudioFlavors(&$flavors, &$curFlavors)
+	{
+		foreach($flavors as $flavor)
+		{
+			if(!in_array($flavor, $curFlavors) && $flavor->hasTag(assetParams::TAG_ALT_AUDIO) && playManifestAction::shouldAddAltAudioFlavors($this->getFormat()))
```

Comment:
check "playManifestAction::shouldAddAltAudioFlavors($this->getFormat()" once before the loop